go/stellar/util.go

```diff
@@ -52,52 +50,20 @@ type OwnAccountLookupCache interface {
 	OwnAccount(ctx context.Context, accountID stellar1.AccountID) (own bool, accountName string, err error)
 }
 
-type ownAccountLookupCacheImpl struct {
-	sync.RWMutex
-	loadErr  error
-	accounts map[stellar1.AccountID]*string
+type ownAccountLookupCacheFromGlobal struct {
+	libkb.Contextified
 }
 
-// NewOwnAccountLookupCache fetches the list of accounts in the background and stores them.
-// Was created before Stellar.accounts, and could probably benefit from using that cache.
 func NewOwnAccountLookupCache(mctx libkb.MetaContext) OwnAccountLookupCache {
```

Comment:
Add a comment like
// `NewOwnAccountLookupCache` was obsoleted and exists only as an interface bridge. Feel free to remove it.